_scripts/prs.py

```diff
@@ -3,6 +3,7 @@
 Requires PyGithub.
```

Comment:
could add `python-dateutil` to this note? or remove the line, as maybe it won't be necessary after the documentation you mentioned is added

Reply:
Yes. I've added requirements.txt which tracks dependencies, but we haven't yet documented this. So for now, I think it's good to mention it here, until it's documented elsewhere.